File: 01_Language/01_Functions/js/chr.js
// https://locutus.io/php/strings/chr/index.html
function chr(codePt) {
    if (codePt > 0xFFFF) { // Create a four-byte string (length 2) since this code point is high
        //   enough for the UTF-16 encoding (JavaScript internal use), to
        //   require representation with two surrogates (reserved non-characters
        //   used for building other characters; the first is "high" and the next "low")
        codePt -= 0x10000
        return String.fromCharCode(0xD800 + (codePt >> 10), 0xDC00 + (codePt & 0x3FF))
    }
    return String.fromCharCode(codePt)
}

console.log("The string ends in escape: " + chr(27))

File: 01_Language/01_Functions/java/src/main/java/com/onevgo/functions/Chr.java
package com.onevgo.functions;

public class Chr {
    public static byte chr(int bytevalue) {
        return (byte) bytevalue;
    }

    public static void main(String[] args) {
        System.out.printf("The string ends in escape: %c", chr(27));
        System.out.println();

        String s1 = new String(new byte[]{chr(240), chr(159), chr(144), chr(152)});
        System.out.println(s1);
    }
}
